src/modules/cron/cron.controller.ts

```diff
@@ -53,4 +62,75 @@ export class CronController {
             throw new Error('Failed to lock users due to an internal server error.');
         }
     }
+
+    @Put('kontext-exceeding')
```

Comment:
I'd use the more descriptive "expired" everywhere

Reply:
done